stackage_snapshot_9.6.2.yaml

```diff
@@ -0,0 +1,10 @@
+resolver: nightly-2023-09-03
+
+packages:
+# See https://github.com/avdv/proto-lens#readme
+- git: https://github.com/avdv/proto-lens
+  commit: 2ab0a8d1ec1f54f3d1a0ecd1a257311389126490
```

Comment:
Is there a path to upstreaming this?

Reply:
This depends on a PR for ghc-source-gen adding support for GHC 9.6 first: https://github.com/google/ghc-source-gen/pull/103

If that is available we could upstream this too.